src/main/java/uk/gov/hmcts/reform/idam/health/probe/ScheduledHealthProbeIndicator.java

```diff
@@ -29,16 +29,21 @@ public ScheduledHealthProbeIndicator(
         this.failureHandling = failureHandling;
         this.freshnessInterval = freshnessInterval;
         this.clock = Clock.systemDefaultZone();
-        taskScheduler.scheduleWithFixedDelay(() -> refresh(), checkInterval);
+        taskScheduler.scheduleWithFixedDelay(this::refresh, checkInterval);
     }
 
     @Override
     public boolean isOkay() {
-        if (status == Status.UNKNOWN) {
-            return this.healthProbe.probe();
+        if (failureHandling == HealthProbeFailureHandling.MARK_AS_DOWN){
+            if (status == Status.UNKNOWN) {
```

Comment:
I think we need to evaluate the probe in any case (irrelevant of the failureHandling) if the status is unknown.

Reply:
My understanding was that when the HealthProbeFailureHandling is set as IGNORE, the method isOkay should always return true, regardless the state. If it's not like this, I'll roll back as it was before.

Reply:
You are right but we should run the probe anyway. Maybe:
```
if (status == Status.UNKNOWN) {
  return this.healthProbe.probe() || failureHandling == HealthProbeFailureHandling.IGNORE;
}
```
Running the probe might produce some logs which help with troubleshooting (if required).

Reply:
OK, I understand. I'll update it